api/envoy/admin/v2alpha/listeners.proto

```diff
@@ -26,7 +26,7 @@ message ListenerStatus {
   // Name of the listener
   string name = 1;
 
-  // The actual local address that the listener is listening on. If a listener was configured
+  // The actual local addresses that the listener is listening on. If a listener was configured
```

Comment:
I think this v2 should not be modified.

Reply:
yea, I modify the wrong file and then forget to revert back.